R/fill_triangles.R

```diff
@@ -41,24 +41,24 @@
 #' retro_rts <- construct_triangles(trunc_rts)
 #' retro_pt_nowcast_mat_list <- fill_triangles(retro_rts)
 #' retro_pt_nowcast_mat_list[1:3]
-fill_triangles <- function(reporting_triangle_list,
+fill_triangles <- function(retro_reporting_triangles,
                            max_delay = min(
-                             sapply(reporting_triangle_list, ncol)
+                             sapply(retro_reporting_triangles, ncol)
                            ) - 1,
                            n = min(
-                             sapply(reporting_triangle_list, nrow)
+                             sapply(retro_reporting_triangles, nrow)
                            ),
                            delay_pmf = NULL) {
   if (is.list(delay_pmf)) { # name as a list and check length of elements
     delay_pmf_list <- delay_pmf
-    if (length(delay_pmf_list) != length(reporting_triangle_list)) {
+    if (length(delay_pmf_list) != length(retro_reporting_triangles)) {
       cli_abort(message = c(
         "List of `delay_pmf` is not the same length as ",
-        "`reporting_triangle_list`."
+        "`retro_reporting_triangles`."
       ))
     }
   } else { # create a list with the same pmf
-    delay_pmf_list <- rep(list(delay_pmf), length(reporting_triangle_list))
+    delay_pmf_list <- rep(list(delay_pmf), length(retro_reporting_triangles))
```

Comment:
internal pt_nowcast_mat_list out of date

Reply:
I don't think I am following here.